com.unity.render-pipelines.universal/Tests/Runtime/RuntimeTests.cs

```diff
@@ -64,12 +68,12 @@ public IEnumerator PipelineSetsAndRestoreGlobalShaderTagCorrectly()
 
     void AssetCheck()
     {
-        //Assert.IsNotNull(currentAsset, "Render Pipeline Asset is Null");
+        //Assert.IsNotNull(currentAssetGraphics, "Render Pipeline Asset is Null");
         // Temp fix, test passes if project isnt setup for Universal RP
-        if (currentAsset == null)
+        if (RenderPipelineManager.currentPipeline == null)
             Assert.Pass("Render Pipeline Asset is Null, test pass by default");
```

Comment:
Minor, related to above. the message should be updated as this is not testing the pipeline asset anymore.